src/test/java/uk/nhs/ctp/controllers/UserControllerTest.java

```diff
@@ -24,16 +29,41 @@
   @Autowired
   private UserController userController;
 
+  @Autowired
+  private UserRepository userRepository;
+
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
+  @After
+  public void cleanup() {
```

Comment:
this is ok but Spring provides 
`@DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)`
Annotated on the class I think this is probably safer?

Reply:
That would have been great, but unfortunately couldn't get it to work. We need to look at not using the actual mysql database in the integration tests (couldn't easily get that working either).